libs/ett/auth/data-access-api/src/lib/oidc/auth-http.service.spec.ts

```diff
@@ -25,17 +25,17 @@ import {
 } from '@energinet-datahub/eo/shared/environments';
 import { lastValueFrom } from 'rxjs';
 
-import { AuthOidcHttp, AuthOidcLoginResponse } from './auth-oidc-http.service';
+import { AuthHttp, AuthOidcLoginResponse } from './auth-http.service';
```

Comment:
Suggestion:

Should we call services for what they are more explicitly?

AuthHttp -> AuthHttpService

Reply:
We do use `Component`, `Pipe`, and `Directive` name suffixes. With those names ensured, it should be clear that everything else is a *service* or rather a *dependency*. Technically, all injectable dependencies are known as *services* in Angular, regardless of whether they are classes, functions, or constant values. Should we also suffix the name of those dependency types with `Service` then? 🤔

Reply:
I see - It sounds fair to leave it as it is to me then, if it is a convention in Angular - I find it nice to be explicit al the way around -> suffixes are used for mostly everything except one. But as you say, if it is injected, then it is technically a service anyways, all good with me :-)

Reply:
If you think about it, the Angular team doesn't suffix the name of any of their services:
- `Router`
- `ActivatedRoute`
- `Location`
- `MatIconRegistry`
- `HttpClient`

Reply:
You're absolutely right, good point :-)